src/lambda/templates/lambdaTemplates.ts

```diff
@@ -20,7 +20,10 @@ export class LambdaTemplates {
         <h3>
             Select a file to use as payload:
         </h3>
-        <input type="file" @change="processFile($event)">
+        <div style="text-overflow: ellipsis;">
+            <button v-on:click="promptForFileLocation">Choose File</button>
+            <span>  {{ selectedFile }}</span>
```

Comment:
could use some spacing. the literal space here isn't going to help (inside/outside of the `<span>` makes no difference).
```suggestion
            &nbsp;  {{ selectedFile }}
```

here's what it looks like currently:

![image](https://user-images.githubusercontent.com/55561878/132927720-902c94bd-7fb7-483f-b8d3-54ff325dfe07.png)